docs-content/general/changelog/changelog-19.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changelog 19 (05/22)
+slug: changelog-19
+---
+
+## Demo Project
+
+Check out [https://app.highlight.io/demo](https://app.highlight.io/demo) to check out our new demo project.
+
+You can play around and see how the Highlight Dashboard works with a fully-featured project with plenty of live data.
```

Comment:
Nit, but "Dashboard" might be misleading here since we also have something called "Dashboards" in the product (https://app.highlight.io/1/dashboards).

Also, this technically isn't a "fully-featured project" since there are no backend logs or errors, but maybe that doesn't matter 🤷

Reply:
Thanks for catching that. I'll tighten it up.